projects/baby_driver/inc/babydriver_msg_defs.h

```diff
@@ -1,11 +1,16 @@
-#pragma once
-
```

Comment:
Looks like your line endings got messed up, the files you've touched have their line endings converted to CRLF while our standard is LF only. To fix this in the future, I think the git command `git config --global core.autocrlf true` should do it; for this PR, you'll have to convert the line endings back into LF. If you're using VSCode you can do it by clicking on the "CRLF" in the bottom-right corner and changing to "LF". I'm sure there's a way to do it for other editors.